examples/go/onboarding.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/dontpanicdao/caigo"
+	"github.com/dontpanicdao/caigo/types"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+// `GET /system/config`
+func GetParadexConfig() (SystemConfigResponse, error) {
+	systemConfigUrl := fmt.Sprintf("%s/system/config", PARADEX_HTTP_URL)
+	response, err := http.Get(systemConfigUrl)
+	if err != nil {
+		return SystemConfigResponse{}, err
+	}
+	responseData, err := io.ReadAll(response.Body)
+	if err != nil {
+		return SystemConfigResponse{}, err
+	}
+	var config SystemConfigResponse
+	err = json.Unmarshal(responseData, &config)
+	if err != nil {
+		return SystemConfigResponse{}, err
+	}
+	return config, nil
+}
+
+// Generate Ethereum public key from Ethereum private key
+func GetEthereumAccount() (string, string) {
+	ethPrivateKey := os.Getenv("ETHEREUM_PRIVATE_KEY")
+	privateKeyBytes, _ := crypto.HexToECDSA(ethPrivateKey)
+	publicKeyECDSA := &privateKeyBytes.PublicKey
+	ethAddress := crypto.PubkeyToAddress(*publicKeyECDSA).Hex()
+	return ethPrivateKey, ethAddress
+}
+
+// TODO: Generate Paradex account from Ethereum private key
+func GenerateParadexAccount(config SystemConfigResponse, ethPrivateKey string) (string, string, string) {
```

Comment:
a little misleading, we are just getting env vars, not generating

Reply:
and the params are not being used

Reply:
Plan is to remove `TODO` and implement stark key derivation from ethereum private key